extras/sequence.c

```diff
@@ -0,0 +1,149 @@
+#include "ggml.h"
```

Comment:
If this serves as a test, I propose moving it to `tests` and reducing the file size by using much shorter test cases.
If this is meant as a runner for users, it should not contain hard-coded strings. But without tokekizer, having a runner that works only with token values does not seem too valuable to me.
Otherwise, I think the file should be removed.

Reply:
This file was designed for benchmarking during development. I think the hardcoded path is ok (it will be modified by anyone who uses it) but I forgot that I included the entire rwkv.cpp source code inside (LMAO).

Reply:
Also since you said the word "tokenizer" https://github.com/LoganDark/rwkv.cpp/tree/rwkv_vocab_v20230424